[2022-11-05 19:48:21,741] [util.py:209]               [INFO]     |====================================================================================================|
[2022-11-05 19:48:21,741] [util.py:216]               [INFO]     |                                     Sci-Fi Collection in Anime                                     |
[2022-11-05 19:48:21,742] [util.py:220]               [INFO]     |====================================================================================================|
[2022-11-05 19:48:21,742] [plex_meta_manager.py:750]  [INFO]     |                                                                                                    |
[2022-11-05 19:48:21,742] [util.py:216]               [INFO]     |=================================== Validating Sci-Fi Attributes ===================================|
[2022-11-05 19:48:21,743] [builder.py:265]            [DEBUG]    |                                                                                                    |
[2022-11-05 19:48:21,743] [builder.py:266]            [DEBUG]    | Validating Method: template                                                                        |
[2022-11-05 19:48:21,743] [meta.py:83]                [DEBUG]    | Value: ordereddict([('name', 'Anime'), ('mal_id', 24), ('poster', 'https://theposterdb.com/api/assets/213635')]) |
[2022-11-05 19:48:21,744] [builder.py:330]            [DEBUG]    |                                                                                                    |
[2022-11-05 19:48:21,744] [builder.py:331]            [DEBUG]    | Validating Method: sync_mode                                                                       |
[2022-11-05 19:48:21,744] [builder.py:335]            [DEBUG]    | Value: sync                                                                                        |
[2022-11-05 19:48:21,744] [builder.py:453]            [DEBUG]    |                                                                                                    |
[2022-11-05 19:48:21,745] [builder.py:454]            [DEBUG]    | Validating Method: collection_order                                                                |
[2022-11-05 19:48:21,745] [builder.py:455]            [DEBUG]    | Value: alpha                                                                                       |
[2022-11-05 19:48:21,745] [builder.py:472]            [DEBUG]    |                                                                                                    |
[2022-11-05 19:48:21,745] [builder.py:473]            [DEBUG]    | Validating Method: mal_genre                                                                       |
[2022-11-05 19:48:21,746] [builder.py:474]            [DEBUG]    | Value: {'genre_id': 24, 'limit': 999}                                                              |
[2022-11-05 19:48:21,746] [builder.py:472]            [DEBUG]    |                                                                                                    |
[2022-11-05 19:48:21,746] [builder.py:473]            [DEBUG]    | Validating Method: url_poster                                                                      |
[2022-11-05 19:48:21,746] [builder.py:474]            [DEBUG]    | Value: https://theposterdb.com/api/assets/213635                                                   |
[2022-11-05 19:48:21,746] [builder.py:472]            [DEBUG]    |                                                                                                    |
[2022-11-05 19:48:21,747] [builder.py:473]            [DEBUG]    | Validating Method: sort_title                                                                      |
[2022-11-05 19:48:21,747] [builder.py:474]            [DEBUG]    | Value: +++_Sci-Fi                                                                                  |
[2022-11-05 19:48:21,747] [builder.py:472]            [DEBUG]    |                                                                                                    |
[2022-11-05 19:48:21,747] [builder.py:473]            [DEBUG]    | Validating Method: sonarr_add_existing                                                             |
[2022-11-05 19:48:21,748] [builder.py:474]            [DEBUG]    | Value: True                                                                                        |
[2022-11-05 19:48:21,820] [builder.py:619]            [INFO]     |                                                                                                    |
[2022-11-05 19:48:21,820] [builder.py:620]            [INFO]     | Validation Successful                                                                              |
[2022-11-05 19:48:21,821] [plex_meta_manager.py:758]  [INFO]     |                                                                                                    |
[2022-11-05 19:48:21,821] [util.py:216]               [INFO]     |==================================== Running Sci-Fi Collection =====================================|
[2022-11-05 19:48:21,821] [plex_meta_manager.py:773]  [INFO]     |                                                                                                    |
[2022-11-05 19:48:21,822] [plex_meta_manager.py:774]  [INFO]     | Sync Mode: sync                                                                                    |
[2022-11-05 19:48:21,822] [builder.py:1275]           [DEBUG]    |                                                                                                    |
[2022-11-05 19:48:21,822] [builder.py:1276]           [DEBUG]    | Builder: mal_genre: {'genre_id': 24, 'limit': 999}                                                 |
[2022-11-05 19:48:21,822] [builder.py:1277]           [INFO]     |                                                                                                    |
[2022-11-05 19:48:21,823] [mal.py:225]                [INFO]     | Processing MyAnimeList Genre ID: 24                                                                |
[2022-11-05 19:48:23,999] [util.py:162]               [DEBUG]    | Traceback (most recent call last):                                                                 |
[2022-11-05 19:48:23,999] [util.py:162]               [DEBUG]    |   File "C:\Plex-Stack\Plex-Meta-Manager\plex_meta_manager.py", line 783, in run_collection         |
[2022-11-05 19:48:23,999] [util.py:162]               [DEBUG]    |     builder.find_rating_keys()                                                                     |
[2022-11-05 19:48:23,999] [util.py:162]               [DEBUG]    |   File "C:\Plex-Stack\Plex-Meta-Manager\modules\builder.py", line 1279, in find_rating_keys        |
[2022-11-05 19:48:24,000] [util.py:162]               [DEBUG]    |     ids = self.gather_ids(method, value)                                                           |
[2022-11-05 19:48:24,000] [util.py:162]               [DEBUG]    |   File "C:\Plex-Stack\Plex-Meta-Manager\modules\builder.py", line 1252, in gather_ids              |
[2022-11-05 19:48:24,000] [util.py:162]               [DEBUG]    |     mal_ids = self.config.MyAnimeList.get_mal_ids(method, value)                                   |
[2022-11-05 19:48:24,000] [util.py:162]               [DEBUG]    |   File "C:\Plex-Stack\Plex-Meta-Manager\modules\mal.py", line 226, in get_mal_ids                  |
[2022-11-05 19:48:24,001] [util.py:162]               [DEBUG]    |     mal_ids = self._genre(data["genre_id"], data["limit"])                                         |
[2022-11-05 19:48:24,001] [util.py:162]               [DEBUG]    |   File "C:\Plex-Stack\Plex-Meta-Manager\modules\mal.py", line 173, in _genre                       |
[2022-11-05 19:48:24,001] [util.py:162]               [DEBUG]    |     raise Failed(f"MyAnimeList Error: No MyAnimeList IDs for Genre ID: {genre_id}")                |
[2022-11-05 19:48:24,001] [util.py:162]               [DEBUG]    | modules.util.Failed: MyAnimeList Error: No MyAnimeList IDs for Genre ID: 24                        |
[2022-11-05 19:48:24,002] [util.py:162]               [DEBUG]    |                                                                                                    |
[2022-11-05 19:48:24,002] [util.py:159]               [ERROR]    | MyAnimeList Error: No MyAnimeList IDs for Genre ID: 24                                             |
[2022-11-05 19:48:24,002] [plex_meta_manager.py:880]  [INFO]     |                                                                                                    |
[2022-11-05 19:48:24,003] [util.py:209]               [INFO]     |====================================================================================================|
[2022-11-05 19:48:24,003] [util.py:216]               [INFO]     |                                     Finished Sci-Fi Collection                                     |
[2022-11-05 19:48:24,003] [util.py:216]               [INFO]     |                                    Collection Run Time: 0:00:02                                    |
[2022-11-05 19:48:24,003] [util.py:220]               [INFO]     |====================================================================================================|
